code/solr/tasks/Solr_Reindex.php

```diff
@@ -65,7 +70,7 @@ protected function doReindex($request)
         // this is for when index names do not match the class name (this can be done by overloading getIndexName() on
         // indexes
         if ($index && !ClassInfo::exists($index)) {
-            foreach(ClassInfo::subclassesFor('SolrIndex') as $solrIndexClass) {
+            foreach(ClassInfo::subclassesFor('SilverStripe\FullTextSearch\Solr\SolrIndex') as $solrIndexClass) {
```

Comment:
Use `SolrIndex::class` instead

I actually can't get `ClassInfo::subclassesFor` to work with SolrIndex. I'm investigating this!